Application/EdFi.Ods.Features/Controllers/CompositeResourceController.cs

```diff
@@ -64,6 +61,19 @@ public class CompositeResourceController : ControllerBase
             _compositeMetadataProvider = compositeMetadataProvider;
             _restErrorProvider = restErrorProvider;
             _isEnabled = apiSettings.IsFeatureEnabled(ApiFeature.Composites.GetConfigKeyName());
+
+            if (apiSettings.IsFeatureEnabled(ApiFeature.MultiTenancy.Value))
```

Comment:
Technically, shouldn't we be using the `ApiFeature.MultiTenancy.GetConfigKeyName()` to be consistent, and maintain a level of separation from the enumeration "Value" being used as the key name?